File: fib.erl
-module(fib).
-export([start/0]).

fib(0) -> 1;
fib(1) -> 1;
fib(2) -> 1;
fib(N) -> fib(N - 2) + fib(N - 1).

start() ->
  io:fwrite("~p~n~p~n~p~n", [fib(44), fib(45), fib(46)]).

File: fib.fs
open System

let rec fib n =
  if n < 3 then 1 else fib (n-2) + fib (n-1)

let testfib n =
  printf "fib(%d) = %d\n" n (fib n)

[<EntryPoint>]
let main argv =
  testfib(44);
  testfib(45);
  testfib(46);
  0
